source/docs/en/guide_github.md

```diff
@@ -1,31 +1,60 @@
 # Using GitHub
-If you want to work on codecheck challenges in your local environment, you can!  
-First you'll need a [github account](https://github.com/join).  
-Then, [connect your GitHub account](https://app.code-check.io/settings/social) to codecheck before you start the challenge.
+Unlike other platforms, if you want to work on codecheck challenges in your local environment,  
+you can!  
 
-## Solving with GitHub
-### Fork challenge
-This menu appears when you click the "Fork" button.
-![イメージ11](images/s11.png)  
-Fill out the repository name and click the "OK" button, then you are going to jump to the generated repository page.  
-Check "Private Repository" if you do not want to publish the repository.
+## Requirements
+- <a href="https://github.com/join" target="_blank"> GitHub account</a>
+- <a href="https://git-scm.com/" target="_blank">Git</a> (If you don't know Git, see <a href="https://try.github.io/" target="_blank">this</a> and <a href="https://git-scm.com/doc" target="_blank">this</a>.)
```

Comment:
Hmm...
Here are my thoughts:

1. This should be super short.  
 - I expect that most users who visit this page already are familiar with git.
 - I also expect there will be a minority of people who want to try programming in local, but don't know git/github.
 - Thus, I want this "if you want to know..." component to not break the structure of the "Requirements" section, so it's easier for the git-familiar users to scan.
 - This means the "if you want to know..." component needs to be very short and concise, and occupies 0 rows.
 - That's why it's expressed as "If you don't know Git, see this and this."
2. The easier solution should be listed first.
 - The "git-scm" link is the full official documentation. This is included because official documentation should always be the central source of truth.
 - However, the "github link" is an interactive tutorial, and is far easier to learn git on. It is also an official resource funded by github.
 - Since this tutorial is an official resource AND it's easier, I think users should be directed to this tutorial at first.